src/main/java/com/scalar/kelpie/config/Config.java

```diff
@@ -268,11 +268,29 @@ public void enableInjector() {
     injectorEnabled = true;
   }
 
+  /**
+   * Sets selected modules enable.
+   *
+   * @param enablePreProcessor if true, sets {@link com.scalar.kelpie.modules.PreProcessor} enable
+   * @param enableProcessor if true, sets {@link com.scalar.kelpie.modules.Processor} enable
+   * @param enablePostProcessor if true, sets {@link com.scalar.kelpie.modules.PostProcessor} enable
+   */
+  public void enableSelectedProcessors(
+      boolean enablePreProcessor, boolean enableProcessor, boolean enablePostProcessor) {
+    if (enablePreProcessor) {
+      enablePreProcessor();
```

Comment:
Just a minor suggestion.
I think we can make it simpler if we have `enableXXXProcessor(boolean)`.

Reply:
Thank you! I modified them.